tests/setup_test_settings.yaml

```diff
@@ -0,0 +1,18 @@
+locations:
```

Comment:
This file should not be included.  It should be generated from the corresponding .in file.

Reply:
It is explicitly included to allow simple downloading of the repo and running a setup check with manually installed databases *without* entering of a build environment and configuring. I have clarified that [section](https://jonasfreimuth.github.io/pigx_docs/pigx-sars-cov-2.html#quick-start) in the docs a bit.

Reply:
Okay, got it.